docs/wiki/[Examples]-Deploy-Custom-Management-Landing-Zone.md

```diff
@@ -0,0 +1,131 @@
+## Overview
+
+This page describes how to deploy Enterprise-scale management resources custom configuration, including guidance on how to apply Log Analytics and Azure Security Center preferences.
+In this example, we take a default configuration and make the following code changes:
+
+### Enable and configure management resources (Level 200)
+- Set and enable the `configure_management_resources`  variable
+- Add parameters for enabling/disabling management resources
+- Add parameters to set location and resource tags
+- Create custom configurations for Log Analytics and ASC
+
+
+The module allows for further configuration of log analytics and azure security center by setting the `configure_management_resources` variable.
+This configuration requires three mandatory variables:
+     location: where the resource group will be deployed
+     advanced:  [tbd]
```

Comment:
We should mark this as experimental for now, and I will update with some of the options which are more likely to remain in the future.

`advanced: [experimental] provides additional options for customisation of your deployment (documentation to follow)`